src/StructuralIdentifiability.jl

```diff
@@ -131,14 +131,36 @@ function assess_identifiability(ode::ODE{P}, funcs_to_check::Array{<: RingElem,1
     return result
 end
 
-function assess_identifiability(de::ModelingToolkit.ODESystem, inputs=[], funcs_to_check=[], p::Float64=0.99)
-    ode, syms, gens_ = PreprocessODE(de, inputs)
+function assess_identifiability(de::ModelingToolkit.ODESystem, p::Float64=0.99)
+    ode, syms, gens_ = PreprocessODE(de)
+    out_dict = Dict()
+    tmp = Dict(param => res for (param, res) in zip(ode.parameters, assess_identifiability(ode, ode.parameters, p)))
+    nemo2mtk = Dict(gens_.=>syms)
+    for (par, res) in pairs(tmp)
+        out_dict[nemo2mtk[par]] = res
+    end
+    return out_dict
```

Comment:
This code seems to repeat the logic an duplicate the code of the more general version of the function. Why not just take the list of parameters of `de` (in the MTK fromat) and pass it as `funcs_to_check` to the method below?

Reply:
I think it should be fixed now